### ISSUE TITLE
Question: Changelog generation for multiple NX projects?

### ISSUE THREAD

**Issue by zeroregard** [jscutlery/semver#574]
I have an application in an NX monorepo. The folder structure looks something like this:

- my-application
    - feature-a
    - feature-b
    - feature-c
    - feature-d
    - shell-application

Where each `feature` is a project containing a `project.json` in which we could add the `version` target.
The problem is that every feature is quite small and I just want to generate a changelog for the entire `my-application`, I don't want individual changelogs for each small `feature`. Is this possible at the moment with semver?

**Comment by edbzn** [comment]
@zeroregard For this use case, I would recommend using the sync-mode with the `--skipProjectChangelog` option, so you have only one root changelog with everything.

**Comment by zeroregard** [comment]
That's the thing though - I have several applications in this mono-repo, and these applications have their own changelogs. It's just that one of the applications has several projects.

**Comment by mvrana-cen81948** [comment]
Yes, this is exactly what i expected as well. I have multiple aplication projects in monorepo and several feature libraries used from that applications. I want changelog from libraries in changelog of application which depends on that libraries.
Option --trackDeps just makes:
```
Dependency Updates
<lib> updated to version #.#.#
```
which is not much informational.
